htsohm/simulation/methane_loading.py

```diff
@@ -72,26 +54,39 @@ def parse_output(output_file):
                 results['ml_excess_volumetric_loading'] = float(line.split()[6])
             elif "Average Host-Host energy:" in line:
                 host_host_line = line_counter + 8
-                results = parse_energy(
-                    output_file, host_host_line, 'host_host', results
-                )
             elif "Average Adsorbate-Adsorbate energy:" in line:
                 adsorbate_adsorbate_line = line_counter + 8
-                results = parse_energy(
-                    output_file, adsorbate_adsorbate_line, 'adsorbate_adsorbate', results
-                )
             elif "Average Host-Adsorbate energy:" in line:
                 host_adsorbate_line = line_counter + 8
-                results = parse_energy(
-                        output_file, host_adsorbate_line, 'host_adsorbate', results
-                )
+            line_counter += 1
+
+    with open(output_file) as origin:
+        line_counter = 1
+        for line in origin:
+            if line_counter == host_host_line:
+                results['ml_host_host_avg'] = float(line.split()[1])
+                results['ml_host_host_vdw'] = float(line.split()[5])
+                results['ml_host_host_cou'] = float(line.split()[7])
+            elif line_counter == adsorbate_adsorbate_line:
+                results['ml_adsorbate_adsorbate_avg'] = float(line.split()[1])
+                results['ml_adsorbate_adsorbate_vdw'] = float(line.split()[5])
+                results['ml_adsorbate_adsorbate_cou'] = float(line.split()[7])
+            elif line_counter == host_adsorbate_line:
+                results['ml_host_adsorbate_avg'] = float(line.split()[1])
+                results['ml_host_adsorbate_vdw'] = float(line.split()[5])
+                results['ml_host_adsorbate_cou'] = float(line.split()[7])
```

Comment:
I think you're just reverting the changes you made earlier with separating this out so I would have preferred this be squashed.